gems/spree/CVE-2013-1656.yml

```diff
@@ -22,3 +22,4 @@ patched_versions:
 related:
   url:
     - https://spreecommerce.com/blog/multiple-security-vulnerabilities-fixed
+    - https://github.com/spree/spree/commit/70092eb55b8be8fe5d21a7658b62da658612fba7
```

Comment:
Extra newline.

Reply:
I think your editor is automatically adding the newline.

Reply:
@v0lck3r GitHub isn't allowing me to use the Suggested change. GitHub is claiming there is no difference between them, even though it's supposed to remove the last newline.

Reply:
@postmodern yeah I have no idea why is that !

Reply:
@postmodern tbh I see no extra line. the file is still [25 lines total with your single reference addition](https://github.com/rubysec/ruby-advisory-db/blob/8853492398cc636f53aff1f704eea4e7c96f2a88/gems/spree/CVE-2013-1656.yml), compared to the[ 24 lines in the original](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/spree/CVE-2013-1656.yml). 
Can you double check please !